src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -529,6 +519,24 @@ public string Base64Fixed(string str)
             }
         }
 
+        private bool IsValidPubKey(string pubKey)
+        {
+            if (pubKey.Length != 66) return false;
+            if (pubKey[0] != '0') return false;
+            if (pubKey[1] != '2' && pubKey[1] != '3') return false;
+
+            for (var i = 2; i < pubKey.Length; i++)
+            {
+                var c = pubKey[i];
+                if (!((c >= '0' && c <= '9') || (c >= 'a' && c <= 'f') || (c >= 'A' && c <= 'F')))
+                {
+                    return false;
+                }
+            }
+
+            return true;
```

Comment:
```suggestion
            try
            {
                _ = ECPoint.Parse(pubkey);
                return true;
            }
            catch
            {
                return false;
            }
```

Reply:
public key can be many length and prefixes.

Reply:
Already removed `IsValidPubKey` method